src/content/documentation/develop/developing-extensions-for-firefox-for-android.md

```diff
@@ -5,15 +5,23 @@ permalink: /documentation/develop/developing-extensions-for-firefox-for-android/
 topic: Develop
 tags: [add-ons, beginner, guide, mobile, webextensions]
 contributors:
-  [rebloor, juraj, mdnwebdocs-bot, ExE-Boss, Ding-Fan, andrewtruongmoz]
+  [caitmuenster, rebloor, juraj, mdnwebdocs-bot, ExE-Boss, Ding-Fan, andrewtruongmoz]
 last_updated_by: rebloor
-date: 2019-06-11 11:14:47
+date: 2019-08-04
 ---
 
 {% capture page_hero_banner_content %}
 
 # Developing extensions for Firefox for Android
 
+::: note alert
+In August 2020, Mozilla will begin rolling out a new, reimagined Firefox for Android experience. The  browser for Android has been rebuilt from the ground up using GeckoView, Mozilla's mobile browsing engine.
```

Comment:
I think we can link `GeckoView` to https://mozilla.github.io/geckoview/ or something better if you know :)

Reply:
Oh yeah good call! 👍